charts/poverty_num_min_china/README.md

```diff
@@ -0,0 +1,4 @@
+# Number of people in Poverty in the World, minus China
+
+##Important
+Instead of 'Poverty Line $/day (2011 PPP)', I chose 'Poverty Line $/day'. Is this fine with Huzaifa. I did not understand the meaning of it and did not want to have the same confusion for the user as well.
```

Comment:
I would leave it as is with the `2011 PPP` because it's a reference value by the world bank. Probably the text around  the graphs should be used to explain the unit used here. https://blogs.worldbank.org/opendata/why-have-2011-ppps-been-revised-and-what-does-it-mean-estimates-poverty